R/distance.R

```diff
@@ -0,0 +1,29 @@
+#install.packages('stringdist')
+#install.packages('proxy')
```

Comment:
Dependencies must be listed in "DESCRIPTION", not installed in R files. Also, please remove all commented out code before opening PR.

Reply:
it is very-very raw draft to prove the concept